lib/template/prop.js

```diff
@@ -39,6 +39,10 @@ export default function prop(item) {
   return `<${xmlnsPrefix(item.namespace)}:${item.name} />`;
 }
 
+export default function putProp(item, value) {
```

Comment:
I think it might be better to handle the case where there's a value in the prop function above?